android/external/MakefileTemplate

```diff
@@ -257,13 +257,27 @@ nettle-install: $(LOCAL)/lib/libnettle.so
 # BOTAN2
 
 botan.src.stamp: ../downloads/botan.src.stamp
-	cp -r ../downloads/botan botan
+	cp -r $(BOTAN_SRC) botan
 	touch $@
 
-botan/Makefile: botan.src.stamp
+## Build jitter
+jitter-build: $(LOCAL)/lib/libjitterentropy.a
+
+$(LOCAL)/lib/libjitterentropy.a:
+	$(MAKE) -C $(JITTER_SRC) jitterentropy-static
+	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-includes PREFIX=$(LOCAL)
+	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-static PREFIX=$(LOCAL)
+
+## Clean jitter
+jitter-clean:
+	-$(MAKE) -C $(JITTER_SRC) clean
+
+botan/Makefile: botan.src.stamp $(LOCAL)/lib/libjitterentropy.a
 	cd botan && \
 		AR="$(AR)" CXX="$(CXX)" \
-			./configure.py --os=android --cc="clang" --cpu="$(BOTAN_ARCH)" --prefix="$(LOCAL)"
+        CXXFLAGS+="-I$(LOCAL)/include -O3" \
+        LDFLAGS+="-L$(LOCAL)/lib -O3" \
+        ./configure.py --os=android --cc="clang" --cpu="$(BOTAN_ARCH)" --prefix="$(LOCAL)"
```

Comment:
* No `-O3` for the linking, it's compiler-only.
* I'd prefer `--extra-cxxflags` to `CXXFLAGS`, `--ldflags` to `LDFLAGS` (less variables).
* The botan build can be shortened (taking this from scripts @thgoebel-planck is using), e.g. the windows version that I happen to have open:

```
    configure.py ^
        --cc=msvc --os=windows --cpu=x86_32 ^
        --without-documentation ^
        --without-sphinx ^
        --with-debug-info ^
        --without-pdf ^
        --without-rst2man ^
        --without-doxygen ^
        --without-boost ^
        --without-openssl ^
        --without-sqlite3 ^
        --extra-cxxflags /I%jitter_include% ^
        --ldflags /LIBPATH:%jitter_lib%
```

After that, explicitly do `make libs` instead of `make` in order to avoid the full CLI build and tests.

Reply:
Done.